code/game/objects/structures/rappel_system.dm

```diff
@@ -132,6 +132,11 @@
 	user.client.perspective = EYE_PERSPECTIVE
 	user.client.eye = target_turf
 
+	if(user.skills.getRating(SKILL_FIREARMS) < SKILL_FIREARMS_DEFAULT)
```

Comment:
this seems pretty smelly relating the firearm skill with an action entirely unrelated to firearms. I won't stop it from being tested, but I don't like this at all.